client-go/cmd/client/client.go

```diff
@@ -47,7 +80,43 @@ func main() {
 		log.Fatalln(errors.Wrap(err, "failed to connect to service"))
 	}
 
-	consumer := consumer.New(conn)
+	ticker := time.NewTicker(time.Millisecond * time.Duration(config.RequestIntervalMS))
+	defer ticker.Stop()
+	done := make(chan bool)
+	for {
+		select {
+		case <-done:
+			return
+		case <-ticker.C:
+			// create a new client for each request
+			consumer := consumer.New(conn)
+			// create random delay to simulate processing load on server
+			// no need for crypto secure rand with seed
+			delay := rand.Intn(config.PublishDelayMaxMS)
+			consumer.HelloWorld(ctx, delay)
+		}
+	}
+}
```

Comment:
Added a basic ticker here to cause the client to stay alive sending requests in order to view the pubsub topic result messages. For a scheduled job in a real production service I would use something like https://github.com/bamzi/jobrunner for this purpose.